src/StreamJsonRpc/StreamJsonRpc.csproj

```diff
@@ -33,6 +35,10 @@
     <PackageReference Include="System.Threading.Tasks.Dataflow" Version="5.0.0" PrivateAssets="compile" />
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
     <PackageReference Include="XliffTasks" Version="1.0.0-beta.19505.1" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="5.0.3">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
```

Comment:
This package is already brought in by our `Directory.Build.props` file.

Reply:
Cool. I will remove it.